Packs/AzureStorage/Integrations/AzureStorage/AzureStorage_test.py

```diff
@@ -325,7 +327,8 @@ def test_test_module_command_with_managed_identities(mocker, requests_mock, clie
     assert client_id and qs['client_id'] == [client_id] or 'client_id' not in qs
 
 
-def test_generate_login_url(mocker):
+@pytest.mark.parametrize('authentication_type', [AUTHORIZATION_CODE, CLIENT_CREDENTIALS_FLOW])
+def test_generate_login_url(mocker, authentication_type):
```

Comment:
the command is relevant only for `authorization code flow`
as in the previous integration: https://github.com/demisto/content/pull/34394#discussion_r1606044150